scripts/update-copyright

```diff
@@ -420,6 +422,7 @@ table.sort(rocks, function(a, b) return a.package:lower() < b.package:lower() en
 
 print("Searching and downloading license texts from rock repos")
 for _, rock in ipairs(rocks) do
+  break
```

Comment:
The break will skip the rest of code, shall we remove the for loop from the source code?